### ISSUE TITLE
Hoobs not accessible

### ISSUE THREAD

**Issue by svharishtech** [hoobs-org/HOOBS#1338]
All  of the sudden, I was not able to connect to any Hoobs connected devices, and then I realized that Hoobs itself was not accessible in my network. The powe light is on, the network port lights are flashing. But hoobs.local is not working. And my network monitoring app is showing Hoobs as offline for past two days. 

Even SSH is not working and just gives a blank page. And hoobs.local gives page not found error. 
I am not sure of the exact current version, but i was on 3.X version. I assume Hoobs did not automatically attempt to upgrade to V4. 

Is there any way i can get back the access to Hoobs?

**Comment by mkellsy** [comment]
Are you using a network cable or wi-if?

See if the HOOBS wi-fi is broadcasting. On your phone go to wi-fi and see if HOOBS shows in the list of available networks.

**Comment by svharishtech** [comment]
I am using it via network cable. It was working  without any issues since almost an year.  I am unable to link this issue with any particular event since I did not access hoobs directly for over a month. The last activity I did was to update Dyson plug-in more than a month ago.

It’s like all of the sudden hoobs refused to be accessible. Both green and orange lights are blinking in the network port of hoobs.

Regards,

Harish SV
